### ISSUE TITLE
Transfered wrong account

### ISSUE THREAD

**Issue by alebalak** [spesmilo/electrum#5428]
Hi, I made a transfer on 21 May but my Bitcoin went to the wrong account...they went to Electrum account. Could you help me ?
<img width="960" alt="bit" src="https://user-images.githubusercontent.com/51857231/59552714-de41d700-8f60-11e9-8561-affa975b4273.png">

**Comment by SomberNight** [comment]
Hi. It is visible on your screenshot that you are using Electrum 4.0.0.
There is no such version at this time, the latest release is 3.3.6.
Sorry to say but looks like you have downloaded a malicious clone.
Those coins are lost.

**Comment by SpyShadow117** [comment]
Had this happen to me yesterday, lost $10, I was trying to transfer my reimbursement from nicehash hack to my bitpay. Once I realized it was a hacked client with a trojan horse, it was too late. Do the following at this time.

*Consider the coins lost
*Re-Install a Clean Copy of Windows
*Consider any wallets you loaded in the client, all private keys compromised. So transfer any funds out of them ASAP.
*Finally, always get a updated client from official website

It sucks, I was planing to get some food that night with it. But we got played. Spread awareness to others about it.
